tpl/barrierless_mess.tpl

```diff
@@ -0,0 +1,20 @@
+!****************************************
+Barrier         RO2_TS          w_1 {bless_prod} ! {bless_rxn}
+  RRHO
+    Stoichiometry               {stoichiometry}
+    Core Rotd
+      File                      {mc_flux.out}
+      SymmetryFactor  8
+      End
+    Frequencies[1/cm]  {n_freq}
+        {freqs fragment 1}
+        {freqs fragment 2}
+
+    ! no tunneling bc no barrier
+
+    ZeroEnergy[kcal/mol]                {prod_energy}
+    ! ElectronicEnergy[kcal/mol]          0
+    ElectronicLevels[1/cm]              1
+          0     2
+End ! end barrierless reaction
+!****************************************
```

Comment:
The template file can be found in /tpl but the user needs to copy the file into their main kinbot or pes run directory and edit everything in the file except for the {prod_energy} and {mc_flux.out} values.